CHANGELOG.rst

```diff
@@ -6,7 +6,9 @@ This document describes changes between each past release.
 1.2.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+**Bug fix**
+
+- Group addons by the biggest guid (#33)
```

Comment:
I see new features like `--api-version` and `--app-version` to CLI  and internal changes like full functional test :)